xonsh/tools.py

```diff
@@ -366,3 +371,13 @@ def suggestion_sort_helper(x, y):
     inx = len([i for i in x if i not in y])
     iny = len([i for i in y if i not in x])
     return lendiff + inx + iny
+
+
+def escape_windows_command_string(s):
+    s = s.replace('%','%%')
+    for c in ('^', '&', '<', '>', '|', "'", '`', ',', ';', '=', '(', ')'):
```

Comment:
Also this might be faster as a regex sub() call... but don't worry about it (ie, non-blocking.)